### PR TITLE
Add remove method to delete an entry from lmdb table

### PR DESCRIPTION
**Pull request by Ram-srini**
1. Added actual API to remove the entry(key->value) from lmdb table.

Signed-off-by: Ramakrishna Srinivasamurthy <ramakrishna.srinivasamurthy@intel.com>

### CODE REVIEW

File: tcs/core/common/python/shared_kv/shared_kv_interface.py [hunk 1]
```diff
@@ -87,14 +87,26 @@ def get(self, table, key):
         return value
 
 #---------------------------------------------------------------------------------------------------
-    def remove(self, table, key):
+    def remove(self, table, key, value=None):
         """
-        Function to remove the value for a key in a lmdb table 
+        Function to remove the kye/value from a lmdb table
```
Comment (manju956) [inline tcs/core/common/python/shared_kv/shared_kv_interface.py]: s/kye/key ?

File: tcs/core/common/packages/db_store/db_store.h [hunk 2]
```diff
@@ -166,6 +189,24 @@ namespace tcf {
...
+         * @return
+         *  TCF_SUCCESS  id->value deleted
+         *  else         failed, d store unchanged
+         */
+        tcf_err_t db_store_del(
+            const std::string& table,
+            const ByteArray& inId,
+            const ByteArray& inValue);
+
     }  /* namespace db_strore */
```
Comment (danintel) [inline tcs/core/common/packages/db_store/db_store.h]: s/db_strore/db_store/

File: tcs/core/tcs_trusted_worker_manager/enclave_wrapper/db_store.h [hunk 3]
```diff
@@ -76,4 +76,21 @@ void db_store_put(
     const std::string& table_b64,
     const std::string& key_b64,
     const std::string& value_b64);
+
+// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+/**
+ * Delets a key->value pair from the database store
```
Comment (manojgop) [inline tcs/core/tcs_trusted_worker_manager/enclave_wrapper/db_store.h]: Delets  -> Deletes

File: tcs/core/tcs_trusted_worker_manager/enclave_wrapper/db_store.cpp [hunk 4]
```diff
@@ -86,4 +86,13 @@ void db_store_put(
...
+void db_store_del(
+    const std::string& table_b64,
+    const std::string& key_b64,
+    const std::string& value_b64) {
+    ByteArray raw_key(key_b64.begin(), key_b64.end());
+    ByteArray raw_value(value_b64.begin(), value_b64.end());
+    tcf_err_t presult = tcf::db_store::db_store_del(table_b64, raw_key, raw_value);
+    ThrowTCFError(presult);
+}
+// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```
[the remark this answers is not in the document]
Comment (Ram-srini) [reply tcs/core/tcs_trusted_worker_manager/enclave_wrapper/db_store.cpp]: Done

File: tcs/core/tcs_trusted_worker_manager/enclave_wrapper/db_store.h (hunk 3 above)
Comment (Ram-srini) [reply tcs/core/tcs_trusted_worker_manager/enclave_wrapper/db_store.h]: Done

**Pull request merged**